tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -61,6 +71,16 @@ Compose kill
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
Are we checking for something here?  Should redis no longer show up in ps?

Reply:
Both redis and web should show up, and I want to see their running status. This is added as extra debug information.